src/app/exams/exams-view.component.html

```diff
@@ -28,9 +28,10 @@
             <planet-markdown-textbox [formControl]="answer"></planet-markdown-textbox>
           </mat-form-field>
           <mat-radio-group *ngSwitchCase="'select'" class="question-list" [formControl]="answer">
-            <mat-radio-button *ngFor="let option of question.choices" [value]="option">
-              {{option.text}}
-            </mat-radio-button>
+            <div *ngFor="let option of question.choices">
+              <mat-radio-button  [value]="option"></mat-radio-button>
```

Comment:
Please remove the double space here

Reply:
@paulbert have made the requested changes. Please review the changes. Thanks!